python/dgl/data/__init__.py

```diff
@@ -11,7 +11,7 @@
 from .ppi import PPIDataset, LegacyPPIDataset
 from .tu import TUDataset, LegacyTUDataset
 from .gnn_benckmark import AmazonCoBuy, CoraFull, Coauthor
-from .karate import KarateClub
+from .karate import KarateClubDataset
```

Comment:
Doesn't it break any example/tutorial?

Reply:
I searched no examples or tutorials using KarateClub class.

Reply:
In case any third-party's using this dataset, I suggest we keep the old named with a warning of the deprecation.